packages/algob/sample-project/js/test/sample-test.js

```diff
@@ -21,6 +21,10 @@ describe("Sample Test", function () {
 		master = new AccountStore(masterBalance);
 		fundReceiver = new AccountStore(minBalance);
 		runtime = new Runtime([master, fundReceiver]);
+		// ## ACCOUNTS loaded from algob.config.js file ##
+		// const cfg = require("../algob.config");
+		// runtime.addAccounts(cfg.networks.default.accounts, 1000000);
```

Comment:
please remove the comments if not needed

Reply:
But shouldn't we need comments to tell user how to get account from algob.config.js?